msg/en_US/csql.msg

```diff
@@ -163,7 +163,9 @@ $ csql help messages
    ;COMmit                      - commit the current transaction.\n\
    ;ROllback                    - roll back the current transaction.\n\
    ;AUtocommit [ON|OFF]         - enable/disable auto commit mode.\n\
-   ;REStart                     - restart database.\n\
+   ;REStart                     - reconnect to the target database in a CSQL session.\n\
```

Comment:
@kisoo-han 
How about "reconnect to the current database in a CSQL session."?
There is no other choice to reconnect target.

Reply:
much better.
I agree with you.

Reply:
Thank you. I will apply it.